std/traits.d

```diff
@@ -8866,7 +8880,7 @@ private template getSymbolsByUDAImpl(alias symbol, alias attribute, names...)
             alias member = __traits(getMember, symbol, names[0]);
 
             // Filtering not compiled members such as alias of basic types.
-            static if (isAliasSeq!member || isType!member)
+            static if (isAliasSeq!member || (isType!member && !isAggregateType!member))
```

Comment:
[User-defined types][1] include both aggregates and enums:

```suggestion
            static if (isAliasSeq!member || (isType!member && !isAggregateType!member && !is(member == enum)))
```

[1]: https://dlang.org/spec/type.html#user-defined-types